File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.cs
using System;

public class Solution {
    public int solution(string binomial) 
    {
        string[] splited = binomial.Split();
        
        int a = Int32.Parse(splited[0]);
        string op = splited[1];
        int b = Int32.Parse(splited[2]);
        
        if(op == "+")
            return a + b;
        else if(op == "-")
            return a - b;
        else // op == *
            return a * b;
    }
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.cpp
#include <string>
#include <vector>
#include <sstream>

using namespace std;

int solution(string binomial) {
    
    stringstream ss(binomial + " ");
    string splited[3]; 
    for (int i = 0; i < 3; ++i)
    {
        std::getline(ss, splited[i], ' ');
    }
    
    int a = stoi(splited[0]);
    string op = splited[1];    
    int b = stoi(splited[2]);

    if (op == "+")      return a + b;
    else if (op == "-") return a - b;
    else                return a * b;
}